waterlog/src/Stylesheets/_segmentLeak.scss

```diff
@@ -1,53 +1,46 @@
 #map-toggle {
     width: 75px;
-    top: -125px;
-    position: relative;
 }
 
 .leakInfo {
     margin: 0 20px;
     padding: 0;
 }
 .low {
-    color: #FFEA00 !important;
+    color: #FFEA00;
```

Comment:
shame rgb comment

Reply:
Standardise your colour, rgb or hex but choose one for the project

Reply:
Sure thanks.